app/services/debates/reopen_service.rb

```diff
@@ -0,0 +1,8 @@
+module Debates
+  class ReopenService < DebateService
+    def call
+      @debate.reopen!
```

Comment:
Currently reopen logic is in two places: `Debate` and `ReopenService`. It should be in one place, in service. It's the same with `CloseService`, which calls method from `Debate`.